### ISSUE TITLE
gnome: update to 3.34.X

### ISSUE THREAD

**Issue by Wychmire** [void-linux/void-packages#15450]
Tracking the progress of updating gnome. Not a request.

Progress in PRs:

- [x] vala: #15444
- [x] glib: #15445
- [x] evolution: #15446 
- [x] vte3: #15448
- [x] gdm: #15477
- [x] sysprof: #15506 (Travis build successful; failed on armv6l-musl due to travis-ci crash)
- [x] mutter: #15530 
- [x] gnome-shell: #15531

Branches exist for
- [x] gnome-builder (waiting on sysprof)

**Comment by thomas-t-w** [comment]
Can you hold back gnome-keyring if possible? Version 3.34 will require mate-session to cary a patch. I think a new mate-session will come out soon that resolves the issue because the patch is upstreamed.

**Comment by Wychmire** [comment]
Sure, I'll avoid updating that package

**Comment by daniel-eys** [comment]
Not sure if already tracked here, but I think there are some rebuilds missing within the evolution update.
The builders are currently complaining with:
```
Inconsistent shlibs:
  libecal-1.2.so.19 (provided by: evolution-data-server; used by: bijiben, gnome-calendar, gnome-panel, gnome-shell, gnome-todo, io.elementary.calendar, libio.elementary.calendar)
```

**Comment by Wychmire** [comment]
I didn't feel my evolution PR was ready to merge (I didn't say anything, however travis was failing to build it), but got merged anyway. 

I'll check those packages and update/revbump them as appropriate.

**Comment by pullmoll** [comment]
The evolution update now causes staging. Updates are required for `libdazzle` (3.34.1): done, `libgweather` (3.34.0) and then the dependencies of `libgweather`: `evolution`, `evolution-data-server`, `gnome-calendar`, `gnome-clocks`, `gnome-initial-setup`, `gnome-panel`, `gnome-settings-daemon`. There may be more which I do not see yet.

**Comment by Wychmire** [comment]
I was planning on doing all the apps which don't require dependency updates in a single PR, which is why I hadn't done anything yet; I was getting all the other applications updated.

`io.elementary.calendar` needs a revbump, no update for that one. What prevents `gnome-todo` from getting a revbump as well? (vs. removal)

**Comment by pullmoll** [comment]
Now I'm stuck with `gnome-shell` because it not only needs `gjs` updated (it is at 1.58.1 now) but also a package `mutter-clutter-5`. I guess this is the updated `mutter-devel`? Please someone take over from here because I'm away for some time now.

**Comment by Wychmire** [comment]
as mentioned in the OP, gnome-shell can't be updated until mutter has been updated, which requires `sysprof` to be updated.

**Comment by pullmoll** [comment]
Then we're stuck with the staging until that point. Too bad.

**Comment by Johnnynator** [comment]
I will work on it. @Wychmire next time just leave the Gnome PR as one, since it seems like everything depends on each other (and gets me to merge it way to early since it looked OK to me)

**Comment by Johnnynator** [comment]
Yes you can mark them [skip ci], I will test them locally.

**Comment by shizonic** [comment]
Thx guys for the awesome work, especially @Wychmire! Really appreciate it.

**Comment by Johnnynator** [comment]
So most things should be updated now.

**Comment by Wychmire** [comment]
Good to hear, I was sort of confused on what was happening after you picked things up

**Comment by Johnnynator** [comment]
> Good to hear, I was sort of confused on what was happening after you picked things up

 Simple, fixed things and updated everything that was needed to be updated :D. You missed a few needed changes here and there in your commits. If you don't see anything else that is mandatory for the core gnome packages than you can probably close this issue.

**Comment by romulasry** [comment]
Update?

**Comment by Wychmire** [comment]
This can probably be closed now. I think pretty much everything is at 3.34.X, though there are even more updates coming in #16892.
